src/app/public/modules/split-view/split-view.component.ts

```diff
@@ -0,0 +1,423 @@
+} from 'rxjs/Subscription';
+
+import 'rxjs/add/operator/take';
+
+import 'rxjs/add/operator/takeUntil';
+
+import 'rxjs/add/operator/takeWhile';
+
+import 'rxjs/add/observable/fromEvent';
+
+import {
+  SkySplitViewBeforeWorkspaceCloseHandler
+} from './types/split-view-before-workspace-close-handler';
+
+import {
+  SkySplitViewMessage
+} from './types/split-view-message';
+
+import {
+  SkySplitViewMessageType
+} from './types/split-view-message-type';
+
+import {
+  SkySplitViewMediaQueryService
+} from './split-view-media-query.service';
+
+import {
+  SkySplitViewListComponent
+} from './split-view-list.component';
+
+import {
+  SkySplitViewWorkspaceComponent
+} from './split-view-workspace.component';
+
+let nextId = 0;
+
+@Component({
+  selector: 'sky-split-view',
+  templateUrl: './split-view.component.html',
+  styleUrls: ['./split-view.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [
+    trigger(
+      'blockAnimationOnLoad', [
+        transition(':enter', [])
+      ]
+    ),
+    trigger(
+      'listEnter', [
+        transition(`void => *`, [
+          style({transform: 'translate(-100%)'}),
+          animate('150ms ease-in')
+        ])
+      ]
+    ),
+    trigger(
+      'workspaceEnter', [
+        transition(`void => *`, [
+          style({transform: 'translate(100%)'}),
+          animate('150ms ease-in')
+        ])
+      ]
+    )
+  ]
+})
+export class SkySplitViewComponent implements OnInit, AfterViewInit, OnDestroy {
+
+  public appSettings: any;
+
+  public splitViewId: string = `sky-split-view-${++nextId}`;
+
+  // Max needs to start as something to allow input range to work.
+  // This value is updated as soon as the user takes action.
+  public listWidthMax = 9999;
+
+  public listWidthMin = 100;
+
+  public listWidthDefault = 320;
+
+  public iteratorNextButtonDisabled = false;
+
+  public iteratorPreviousButtonDisabled = false;
+
+  @Output()
+  public iteratorNextButtonClick = new EventEmitter<void>();
+
+  @Output()
+  public iteratorPreviousButtonClick = new EventEmitter<void>();
+
+  public set isListVisible(value: boolean) {
+    this._listVisible = value;
+  }
+
+  public get isListVisible() {
+    return !this.isMobile || this._listVisible;
+  }
+
+  public get workspaceVisible() {
+    return !this.isMobile || !this._listVisible;
+  }
+
+  @Input()
+  public set listWidth(value: number) {
+    if (value) {
+      this._listWidth = value;
+      this.updateBreakpoints();
+    }
+  }
+  public get listWidth() {
+    if (this.isMobile) {
+      return undefined;
+    } else {
+      if (this._listWidth > this.listWidthMax) {
+        return this.listWidthMax;
+      } else if (this._listWidth < this.listWidthMin) {
+        return this.listWidthMin;
+      } else {
+        return this._listWidth || this.listWidthDefault;
+      }
+    }
+  }
+
+  @Input()
+  public messageStream = new Subject<SkySplitViewMessage>();
+
+  @Input()
+  public ariaRole: string;
```

Comment:
For the sake of documenting the user guidelines, what would be a realistic value for the `ariaRole`, here?

Reply:
@Blackbaud-MattGregg do you have any feedback on this? What would be a good `ariaRole` for a split view?

Reply:
@Blackbaud-AlexKingman For the splitview, I think what we'd want is for each panel to be a region on the page to act as landmarks. This makes it easy for navigation. https://www.w3.org/TR/wai-aria-practices/examples/landmarks/region.html
Teams would need to be able to set a property for aria-label which would be the name for the region.

Reply:
@Blackbaud-MattGregg I can easily add the aria-labels to each panel. Thanks for the feedback! One last question - do we want to enforce the role of `region`, or is that something we just outline in the docs?

Reply:
In the meantime, I've updated roles & labels to be applied to each panel.

Reply:
Let's talk more about this offline.

Reply:
Yes. I think each panel would be a region. These are the main sections of the page or form which would be relevant to have in the page outline and support someone navigating between them. Teams would need to put in aria-label values that make sense for their use and each needs to be unique (e.g., List of invoices, Invoice selected to approve)